rpc/rest/security/model/http-security/httpsecurity.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto3";
+
+// Package http_security provides basic definition of user and permission group
+package http_security;
```

Comment:
After yesterday's call, we know that this will need to be more generic and serve other uses than just HTTP. We should rename it to `AccessSecurity`.